contributors/design-proposals/encryption.md

```diff
@@ -0,0 +1,204 @@
+# Encryption
```

Comment:
Thanks, I think we're all in agreement to get this done and we only have a few weeks left before 1.7 code freeze.